### ISSUE TITLE
Disabled Input and styles

### ISSUE THREAD

**Issue by iRoachie** [react-native-training/react-native-elements#1756]
**Is your feature request related to a problem? Please Describe.**
There's no way to disable the Input component or customize the appearance when it is disabled.

**Describe the solution you'd like**
A `disabled` prop which is a boolean, along with various props for managing styles when disabled.

**Describe alternatives you've considered**
The only alternative right now is doing it yourself managing your own styles.

E.g.

```jsx
<Input inputStyle={disabled ? styles.disabled : styles.normal}/>
```

**Additional context**
Similar issue #1715

**Comment by saxenanihal95** [comment]
I want to work on this, one question do we provide some default disabled style ?

**Comment by iRoachie** [comment]
I think we should.

**Comment by mayconmesquita** [comment]
This would be nice!

**Comment by lbelavoir** [comment]
@saxenanihal95 have you started on this one yet ?

**Comment by saxenanihal95** [comment]
no not right now

**Comment by jdmg94** [comment]
can we get a disabled state for other inputs like checkboxes and buttons please

**Comment by iRoachie** [comment]
@jdmg94 Buttons already have it! Should only be checkboxes left.

**Comment by iRoachie** [comment]
Merged 💯Will be live in 1.2.0
